libres/lib/enkf/local_obsdata.cpp

```diff
@@ -126,20 +127,37 @@ bool local_obsdata_has_node(const local_obsdata_type *data, const char *key) {
     return hash_has_key(data->nodes_map, key);
 }
 
-active_list_type *
+ActiveList *
 local_obsdata_get_copy_node_active_list(const local_obsdata_type *obsdata,
                                         const char *obs_key) {
     local_obsdata_node_type *obsdata_node = local_obsdata_get(obsdata, obs_key);
-    active_list_type *active_list =
-        local_obsdata_node_get_copy_active_list(obsdata_node);
+    auto *active_list = local_obsdata_node_get_copy_active_list(obsdata_node);
     return active_list;
 }
 
-active_list_type *
+ActiveList *
 local_obsdata_get_node_active_list(const local_obsdata_type *obsdata,
                                    const char *obs_key) {
     local_obsdata_node_type *obsdata_node = local_obsdata_get(obsdata, obs_key);
-    active_list_type *active_list =
-        local_obsdata_node_get_active_list(obsdata_node);
+    auto *active_list = local_obsdata_node_get_active_list(obsdata_node);
     return active_list;
 }
+
+
+namespace {
+ActiveList &get_active_list(py::handle obj, const std::string &name) {
+    auto *node = reinterpret_cast<local_obsdata_type *>(
```

Comment:
Could be replaced with `ert::from_cwrap<local_obsdata_type>(obj)`

Reply:
OK